lib/include/tick/linear_model/model_poisreg.h

```diff
@@ -11,37 +11,51 @@
 
 // TODO: labels should be a ArrayUInt
 
-enum class LinkType { identity = 0, exponential };
+enum class LinkType : uint16_t { identity = 0, exponential };
```

Comment:
Is `uint16_t` needed? If not I think it just makes the code more frightening.

Reply:
This is good practice to specify the "underlying" type

[See](https://github.com/X-DataInitiative/tick/pull/199/files/dc5b9b3a14cd61ee368a27472ad29bccfae02902#diff-3279f659de5f919d107b73e16c0dce09R16)

Reply:
Alright!